sources/metricdog/src/metricdog.rs

```diff
@@ -105,8 +111,29 @@ impl Metricdog {
 
     /// Sends a notification to the metrics url that boot succeeded.
     pub(crate) fn send_boot_success(&self) -> Result<()> {
-        // timeout of 3 seconds to prevent blocking the completion of mark-boot-success
-        self.send("metricdog", "boot_success", None, Some(3))?;
+        let mut values = HashMap::new();
+        // Note whether this is the first boot or not
+        values.insert(
+            "is_first_boot".to_string(),
```

Comment:
Should this be a constant? Same for the others below.

Reply:
I don't think they have to be constants since the values are only going to be referenced here.